packages/quantic/cypress/integration/facets/facet/facet.cypress.ts

```diff
@@ -0,0 +1,322 @@
+          }
+
+          describe('verify rendering', () => {
+            before(selectSearchResult);
+
+            Expect.numberOfIdleCheckboxValues(defaultNumberOfValues - 1);
+            Expect.numberOfSelectedCheckboxValues(1);
+            Expect.displayMoreMatchesFound(false);
+            Expect.displayNoMatchesFound(false);
+            Expect.displayShowMoreButton(true);
+            Expect.displaySearchInput(true);
+            Expect.displaySearchClearButton(false);
+          });
+
+          describe('verify analytics', () => {
+            before(selectSearchResult);
+
+            Expect.logFacetSelect(defaultField, 0);
+          });
+        });
+
+        describe('when searching for a value that returns a single result', () => {
+          describe('verify rendering', () => {
+            before(searchForSingleValue);
+
+            Expect.numberOfIdleCheckboxValues(1);
+            Expect.numberOfSelectedCheckboxValues(0);
+            Expect.displayMoreMatchesFound(false);
+            Expect.displayNoMatchesFound(false);
+            Expect.displaySearchClearButton(true);
+          });
+        });
+
+        describe('when searching for a value that returns no results', () => {
+          const query = 'this facet value does not exist';
+
+          function searchForInvalidValue() {
+            setupWithValues();
+            FacetSelectors.searchInput().type(query);
+          }
+
+          describe('verify rendering', () => {
+            before(searchForInvalidValue);
+
+            Expect.numberOfIdleCheckboxValues(0);
+            Expect.numberOfSelectedCheckboxValues(0);
+            Expect.displayMoreMatchesFound(false);
+            Expect.displayNoMatchesFound(true);
+            Expect.noMatchesFoundContainsQuery(query);
+            Expect.displaySearchClearButton(true);
+          });
+        });
+      });
+    });
+
+    describe('when collapsing a facet', () => {
+      function collapseFacet() {
+        setupWithValues();
+        FacetSelectors.collapseButton().click();
+      }
+
+      describe('verify rendering', () => {
+        before(collapseFacet);
+
+        Expect.labelContains(defaultLabel);
+        Expect.displayExpandButton(true);
+        Expect.displaySearchInput(false);
+        Expect.numberOfIdleCheckboxValues(0);
+        Expect.displayShowMoreButton(false);
+      });
+
+      describe('when expanding a facet', () => {
+        function expandFacet() {
+          collapseFacet();
+          FacetSelectors.expandButton().click();
+        }
+
+        describe('verify rendering', () => {
+          before(expandFacet);
+
+          Expect.labelContains(defaultLabel);
+          Expect.displayCollapseButton(true);
+          Expect.displaySearchInput(true);
+          Expect.numberOfIdleCheckboxValues(defaultNumberOfValues);
+          Expect.displayShowMoreButton(true);
+        });
+      });
+    });
+  });
+
+  describe('with custom field and label', () => {
```

Comment:
repeating test, as field and label are something you must provided

Reply:
The test is repeated if you assume that the component is really reading the field and label from these properties. By testing with a different set of values, we are certain the component is not using any default value behind the scene.

Reply:
it's not necessary, if you change only field/label in this case, more likely you test integration with searchAPI side. If you're afraid of we are using 'default field' too much, just include custom field/ label with other tests like 'custom numberOfValue', we will save 1s for the test :P

Reply:
I personally prefer to test options separately, but since those are very simple options, I'm ok merging them.